Functions.Templates/Templates/SqlInputBinding-JavaScript/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "defaultFunctionName": "SqlInputBinding",
+  "description": "$SqlInputBinding_description",
+  "name": "SQL Input Binding",
+  "language": "JavaScript",
+  "category": [
+    "$temp_category_core",
+    "$temp_category_dataProcessing"
+  ],
+  "categoryStyle": "sql",
+  "enabledInTryMode": false,
```

Comment:
do we know what TryMode is?

Reply:
no, will have to check with Naren, but I have seen couple if examples and all of them have this value as false.

Reply:
Probably https://www.tryfunctions.com/ng-min/try?trial=true (although that's never worked for me...)

Reply:
Confirmed per the docs : https://github.com/Azure/azure-functions-templates#template-files-1

`    "enabledInTryMode": true,                  // Should this template be available in try mode: https://tryfunctions.com/ng-min/try?trial=true`

Reply:
So we should leave this as false for now I think given that most users won't have a DB available that would be accessible by that. We can follow up with enabling it later if we find out a good way to do that.